Sources/VisitProposalExtension.swift

```diff
@@ -15,4 +15,12 @@ public extension VisitProposal {
         }
         return .default
     }
+    
+    var viewController: String {
+        if let viewController = properties["view-controller"] as? String {
+            return viewController
+        }
+        
+        return "VisitableViewController"
```

Comment:
I don't think we should promote using class names as identifiers. It makes the approach brittle if there are code refactorings. For example, `"web"` would be a fine identifier, `"web-modal"` for a view controller specific to handling web modals, etc. 

Should we even have a default here? We could either make this optional or throw an exception and promote using a global rule at the top of your path config like:
```json
"rules": [
    {
      "patterns": [
        ".*"
      ],
      "properties": {
        "context": "default",
        "presentation": "default",
        "view-controller": "web",
        "..."
      }
    },
    {
        "more specific rules"
    }
]
```

Reply:
> We could either make this optional or throw an exception and promote using a global rule at the top of your path config like

The key question is whether we want to turn `"view-controller"` a required key.
- If we do, then throwing an exception seems a bit harsh but it'll be fine if we turn this into a function instead of a variable. 
- If we don't, then a default makes the most sense. We can easily add protocol conformance for VisitableViewController via an extension right now. I originally didn't think of it because we're not in Turbo, but we can set it here and then incorporate it as part of the class in the future.

I don't have enough experience with configuration files, so I don't think I can make the call here.

Reply:
I'm against throwing an exception as this could break backwards compatibility. If a `"view-controller"` value is received but the app can't match it then it should default to the default controller.

I like having a default value, especially because you already need a config file for all of the 15+ navigation routes to work. This also might help clear up a direction for #50 if we have one that "every project should include".

Reply:
I've extended `VisitableViewController` so it now conforms to `PathConfigurationIdentifiable`. This gives us a default value that is not tied to the class name.